src/quicklist.c

```diff
@@ -136,6 +136,37 @@ quicklist *quicklistNew(int fill, int compress) {
     return quicklist;
 }
 
+void quicklistRepr(unsigned char *ql)
+{
+    quicklist *quicklist  = (struct quicklist*) ql;
+
+    printf("count=%ld\n", quicklist->count);
+    printf("len=%ld\n", quicklist->len);
+    printf("fill=%d\n", quicklist->fill);
+    printf("compress=%d\n", quicklist->compress);
+    printf("bookmark_count%d\n", quicklist->bookmark_count);
+    quicklistNode* node = quicklist->head;
+    int i=0;
+    while(node != NULL) {
+        printf("===========================================\n");
+        printf("============quicklist node(%d)============\n", i++);
+        printf("===========================================\n");
```

Comment:
"prints the actual content of the node, or just their metadata (i.e. it won't call ziplistRepr, and it wont print the actual string for sds, but rather just the length)." 
what do you mean by just the length ? for a "plain" node I will print the value and for ziplist node I will only print number of entries ?

Reply:
i mean that if that flag is off, you only print all the metadata you have on the node, without calling ziplistRepr or printing the value of the plain string.
this can be the number of records, the size (of the string or ziplist), the container type, and compression.
if the flag is on, you print all the above, but also print the data (ziplistRepr or print the actual string/buffer)